src/store/form/schemas.json

```diff
@@ -269,7 +269,7 @@
                 "ui:widget": "textarea"
             },
             "q5": {
-                "ui:widget": "textarea"
+                "ui:widget": "linkarea"
```

Comment:
Are we only going to linkify q5? or other questions too?

Reply:
@MYKatz let's go ahead and linkify all the text areas in case people drop links in them. Also, can we ensure all links have target="_blank" to ensure new tab so people don't lose the review their in the middle of?